### PR TITLE
Revert pr121 arch fix

### PR DESCRIPTION
**Pull request by eskaur**
Note how both commits in this PR are green (it now works with both `archlinux/base` and `archlinux/archlinux:base`), so the second commit is not strictly necessary. However, I added the second commit due to the discussion in #122 .
I'll let you Arch experts decide what's best.

### CODE REVIEW

**Comment by knzivid** [comment]
Let's stick to arch's arch image

**Comment by knzivid** [comment]
Like I previously mentioned, we are already rolling; why not roll with upstream?

**Comment by eskaur** [comment]
> Like I previously mentioned, we are already rolling; why not roll with upstream?

Guess I need approval from a "code owner". @nedrebo ?

**Review by nedrebo — APPROVED** [review]

**Pull request merged**
